apps/etterlatte-saksbehandling-ui/client/src/components/person/brev/handlinger/BrevUtgaar.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from 'react'
+import { useApiCall } from '~shared/hooks/useApiCall'
+import { Alert, BodyShort, Button, HStack, Modal, Textarea, VStack } from '@navikt/ds-react'
+import { ArrowUndoIcon } from '@navikt/aksel-icons'
+import { isPending, mapResult } from '~shared/api/apiUtils'
+import { BrevStatus, IBrev } from '~shared/types/Brev'
+import { differenceInWeeks } from 'date-fns'
+import { markerBrevSomUtgaar } from '~shared/api/brev'
+import { ApiErrorAlert } from '~ErrorBoundary'
+
+const EN_UKE = 1
+
+/**
+ * Sjekker om brevet er ferdigstilt eller journalført og over 1 uke gammelt.
+ * Dette er som oftest et tegn på at noe har gått galt i flyten
+ **/
+const kanMarkereSomUtgaar = (brev: IBrev) =>
+  [BrevStatus.FERDIGSTILT, BrevStatus.JOURNALFOERT].includes(brev.status) &&
+  differenceInWeeks(new Date(), brev.statusEndret) >= EN_UKE
+
+/**
+ * Handling som kan brukes i tilfeller hvor et brev ikke lenger er relevant/aktuelt.
+ **/
+export const BrevUtgaar = ({ brev }: { brev: IBrev }) => {
+  const [isOpen, setIsOpen] = useState(false)
+  const [kommentar, setKommentar] = useState<string>()
+  const [brevUtgaarResult, apiBrevUtgaar] = useApiCall(markerBrevSomUtgaar)
+
+  if (!kanMarkereSomUtgaar(brev)) {
+    return null
+  }
+
+  const utgaar = () => {
+    if (!kommentar) return
```

Comment:
Blir det feilmelding i registrering hvis ikke?

Reply:
Nei, bare får ikke sendt. Har ikke tenkt til å lage alt for mye greier rundt dette siden hele løsningen er midlertidig.